### DISCUSSION TITLE
PJSはPM領域のどの部分を担当するのかの分類が最初にあると良いのではないか？

### DISCUSSION THREAD

**Discussion by CP-koshikawa** [copilot-jp/project-sprint#383] [Q&A]
TheoriesとPracticesに大きな違和感はないし、この部分できてなかったなというものもありました。
ただ、クライアントワークのPM支援全体となるともっと細かい作業があると思います。
PJSはPMのどの部分を担当するのかの分類が最初にあると理解が深まるし、欲しい情報が得られるかの判断基準になるかと思いました

**Comment by MasaruSaito** [comment]
PMBOKとの比較表みたいなものがあるとよいのかと思いました

**Comment by motoi** [comment]
PJM作業の表現とPJSがどのような関係かを考えたうえでどう表現するかを考える必要があるとおもいました。
テンションとしては読むきっかけが欲しいということなのかなと理解しました。

**Reply by motoi** [reply]
「プロジェクトマネジメント」という記述があるのでPJMとの関係がきになるのかも。

**Comment by CP-koshikawa** [comment]
＜結論＞
こちらのDisscussionのIssuesを立てる【こころさん】
